customTests/azure_cpu_stream.nhc

```diff
@@ -119,7 +119,7 @@ function check_cpu_stream(){
         result=${values[$i]}
         dbg stream_${result_labels[$i]}: $result
         if (( $(echo "$result < $EXP_PERF" | bc -l) )); then
-            die 1 -e "\t **Fail** Stream test failed on at least one function less than expeted value: $EXP_PERF. Stream Results: $stream"
+            die 1 -e "\t **Fail** Stream test failed on at least one function less than expeted value: $EXP_PERF. Stream Results: $stream. FaultCode: NHCNA"
```

Comment:
This is an actual failure

Reply:
Labeling it as Generic Failure for now as we discuss plans to create an impact code for it